data-raw/impeachment-polls/process_data_set_Fatima.R

```diff
@@ -0,0 +1,29 @@
+library(dplyr)
```

Comment:
I don't think we need this file at all, since the previous one does the same thing?

Reply:
I don't think @rudeboybert wants these files in a subdirectory.